### ISSUE TITLE
libsrtp-2.0.0 # make runtest  ./rtpw: couldn't open file /usr/share/dict/words make: *** [runtest] Error 255

### ISSUE THREAD

**Issue by mariarti** [cisco/libsrtp#299]
```
# make runtest
Build done. Please run 'make runtest' to run self tests.
running libsrtp2 test applications...
crypto/test/cipher_driver -v >/dev/null
crypto/test/kernel_driver -v >/dev/null
test/rdbx_driver -v >/dev/null
test/srtp_driver -v >/dev/null
test/roc_driver -v >/dev/null
test/replay_driver -v >/dev/null
test/dtls_srtp_driver >/dev/null
cd test; /root/asterisk/libsrtp-2.0.0/test/rtpw_test.sh >/dev/null	
./rtpw: couldn't open file /usr/share/dict/words
make: *** [runtest] Error 255
```
centos 6.9

**Comment by paulej** [comment]
I think that last error is just due to the "spell" or "words" package missing from the machine.

**Comment by pabuhler** [comment]
This should be fixed now by #300
